script/fix

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# script/fix: Run automated format fixers
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+[ -z "$DEBUG" ] || set -x
+
+echo "==> Linting in fixer mode..."
```

Comment:
Should this line only appear if we're going to run the linters, i.e. inside the `if [ -n "$CI" ]` ?